docs/README.md

```diff
@@ -0,0 +1,111 @@
+## Overview
+`kube-bench` runs YAML specifications of the CIS Kubernetes Benchmarks against a kubernetes cluster.
+
+## Architecture
+benchmark spec  --> kube-bench <-- configuration files
```

Comment:
I'd actually say the benchmark spec -> config files -> kube-bench -> results (rather than a direct connection between benchmark spec and kube-bench) - wdyt?

Reply:
> Is the idea here to have a separate docs file, or is this supposed to be a WIP on the main README? If the latter, let's just do PRs against the README so it's easy to read the diffs.
> 
> Having said that, I quite like the idea of separating the docs for config file syntax into a separate file that's linked from the main README.

The idea is to have a separate document file(s) linked to from README.md. This way we can keep the README short and sweet and point users to details in the `docs/` directory.

WDYT?

Reply:
That sounds good to me